github_actions/spec/dependabot/github_actions/update_checker_spec.rb

```diff
@@ -381,11 +384,11 @@
       end
     end
 
-    context "given a dependency with multiple git refs", :vcr do
```

Comment:
Where these `vcr` cassettes actually unused, or just outdated?

I'm personally fine dropping them, but if I'm reading this correctly it means these tests now require internet access to run whereas previously they didn't... 

Again, I'm fine with this change, just wanted clarification since commit message says dropping _unused_ cassettes...

Reply:
Extracted and improved this particular change as #7233.